src/main/java/cms/gov/madie/measure/dto/TestCaseLock.java

```diff
@@ -0,0 +1,27 @@
+package cms.gov.madie.measure.dto;
+
+import java.time.Instant;
+
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.index.Indexed;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Document(collection = "testCaseLock")
+public class TestCaseLock {
+  private String measureId;
+  @Id private String testCaseId;
```

Comment:
this may not be an issue but just raising the point if anyone see any issue- is there a chance of test case id being not unique across the measures..maybe import or API usage?